third_party/parallel_hashmap/download.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+cd $(dirname $0)
+
+revision=d88c5e15079047777b418132ece5879e7c9aaa2b
+
+for file in phmap.h phmap_base.h phmap_bits.h phmap_config.h phmap_fwd_decl.h phmap_utils.h
```

Comment:
I would suggest respecting the upstream file layout, even if only download a few of them. This will be easier long-term for maintenance, and allows adding a few more things (e.g. the `phmap_{gdb,lldb}.py` files might be handy while debugging Ninja).

In this case, this would mean, copying the headers to `third_party/parallel_hashmap/parallel_hashmap` and adding `third_party/parallel_hashmap` instead of `third_party` to the include directory. Wdyt?

Reply:
Thanks, I updated to use upstream layout.